ompi/mca/coll/libnbc/coll_libnbc.h

```diff
@@ -75,7 +75,7 @@ struct ompi_coll_libnbc_component_t {
     opal_free_list_t requests;
     opal_list_t active_requests;
     int32_t active_comms;
-    opal_atomic_lock_t progress_lock; /* protect from recursive calls */
+    bool in_progress;                 /* protect from recursive calls */
```

Comment:
imho, ```in_progress``` should rather be a static variable of ```coll_libnbc_component.c``` since it is not used anywhere else